inst/htmlwidgets/Leaflet.glify/addGlifyPoints.js

```diff
@@ -49,32 +49,37 @@ LeafletWidget.methods.addGlifyPoints = function(data, cols, popup, opacity, size
   map.layerManager.addLayer(pointslayer.glLayer, null, null, group);
 */
 
-    //var dat = JSON.parse(points);
-    //if (popup_var) var pop = JSON.parse(popups);
-    var pointslayer = L.glify.points({
-      map: map,
-      click: function (e, point, xy) {
-        var idx = data.findIndex(k => k==point);
-        //set up a standalone popup (use a popup as a layer)
-        if (map.hasLayer(pointslayer.glLayer)) {
+  var pointslayer = L.glify.points({
+    map: map,
+    click: function (e, point, xy) {
+      var idx = data.findIndex(k => k==point);
```

Comment:
@trafficonese  Isn't `idx` here the `id` for the click event that we are looking for?

Reply:
Yes for points it would be be easy (actually it would be `idx+1`), but for lines/shapes there is currently no `idx` when popup is not NULL or TRUE. I'll rearrange the code a bit, so that `idx` is always calculated.